packages/eslint-config-airbnb-base/index.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  extends: [
+    './legacy',
+    './rules/es6',
+  ].map(require.resolve),
+  parserOptions: {
+    ecmaVersion: 7,
+    sourceType: 'module',
+  },
+  rules: {
+    strict: 2,
```

Comment:
In the original [base.js](https://github.com/airbnb/javascript/pull/829/files#diff-ca246d3b4ba146243647f872c0e7e8d1), `rules` was empty and `parserOptions` did not exist (`ecmaVersion: 7` here makes no sense for a "base").

I would expect a project that extended `airbnb/base` converting to `airbnb-base` to be as simple as shuffling npm dependencies:

```bash
npm ls eslint-config-airbnb
# └── eslint-config-airbnb@7.0.0
cat .eslintrc.json
# { "extends": "airbnb/base" }
./node_modules/.bin/eslint .
# no errors

npm i -D eslint-config-airbnb-base
npm rm -D eslint-config-airbnb eslint-plugin-jsx-a11y eslint-plugin-react

echo '{ "extends": "airbnb-base" }' > .eslintrc.json
./node_modules/.bin/eslint .
# several dozen errors of type "strict" and "global-require"
```

I was heartened to see a base module was extracted without the unfortunate `peerDependencies` on things I wasn't ever going to use (I'm writing a native node v4+ `hapi` server application). I can of course overwrite the offending options, but it really shouldn't be necessary.

```
{
    "extends": "airbnb-base",
    "parserOptions": {
        "ecmaVersion": 6,
        "sourceType": "script"
    },
    "rules": {
        "global-require": 0
    }
}
```

I'm not sure where `global-require` popped up, since it wasn't complaining in `eslint-config-airbnb/base` before...